script/Deployer.sol

```diff
@@ -92,11 +99,12 @@ contract Deployer is Script {
     }
 
     function removeDeployerAccess(address router, address deployer) public {
-        RouterLike(router).deny(deployer);
+        AuthLike(router).deny(deployer);
         root.deny(deployer);
         investmentManager.deny(deployer);
         poolManager.deny(deployer);
         escrow.deny(deployer);
+        userEscrow.deny(deployer);
         gateway.deny(deployer);
         pauseAdmin.deny(deployer);
         delayedAdmin.deny(deployer);
```

Comment:
Looks like router is also missing here?

Reply:
the router is the first one that is denied

Reply:
Ah true!